config/nrfconnect/chip-module/Kconfig

```diff
@@ -72,3 +72,228 @@ config CHIP_DEBUG_SYMBOLS
 	default y
 	help
 	  Build the application with debug symbols.
+
+config CHIP_FACTORY_DATA_ENABLE
+	bool "Enables Factory Data support"
+	default n
+	help
+	  Enables generation of factory data after building.
+	  It requires factory_data partition to exist in the partition manager 
+	  configuration file pm_static.yml.
+	  As a result a new output file factory_data.hex will be created.
+
+if CHIP_FACTORY_DATA_ENABLE
+
+# Factory data definitions
+config CHIP_FACTORY_DATA_PARTITION_ADDRESS
```

Comment:
Why do we require a user to define those in Kconfig? We can take it from the partition manager output not to duplicate the configs.

Reply:
hmm, You're right, this address should be connected with the partition manager.